### PR TITLE
Platform-1103 switch to Universal Analytics

### PR DESCRIPTION
**Pull request by harnash**
Since we do switch on Wikia/app side to only use Universal Analytics this change does the same thing on Mercury side. All Google Analytics (legacy) code was removed and new Universal Analytics code was provided.

### CODE REVIEW

**Comment by hakubo** [comment]
LGTM

File: test/specs/front/scripts/mercury/modules/Trackers/UniversalAnalytics.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+		window.ga = function () {
+			queue.push(Array.prototype.slice.call(arguments));
+		};
+		this.queue = queue;
+
+
+		/**
+		 * Checks if two objects are equal (recurisvely)
+		 */
+		this.objectEquals = function (x, y) {
```
Comment (hakubo) [inline test/specs/front/scripts/mercury/modules/Trackers/UniversalAnalytics.js]: hmm this looks like it could be abstracted to some helpers files

Comment (harnash) [inline test/specs/front/scripts/mercury/modules/Trackers/UniversalAnalytics.js]: Done in db85edf.

**Comment by hakubo** [comment]
Tests are passing
Tested locally and events are sent to GA

**Pull request merged**